src/test/java/com/google/sps/SectionTest.java

```diff
@@ -0,0 +1,110 @@
+package com.google.sps;
+
+import com.google.sps.data.Section;
+import com.google.sps.data.TimeRange;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class SectionTest {
+
+  private static final TimeRange MONDAY_10AM = 
+      TimeRange.fromStartDuration(TimeRange.MONDAY, 10, 0, 60);
```

Comment:
Maybe might be worth writing a variable like `private static final int DURATION_1_HOUR = 60;` to avoid writing 60